CONTRIBUTING.md

```diff
@@ -1,81 +1,141 @@
+
+1. Open an issue for your pull request first (unless already opened).
+2. Create your own fork of RSQKit repository (even if you have write access to the RSQKit repository) and set it up to render RSQKit pages as a Web site. An even better 
+way is to create a branch in your fork for each significant change (pull request), as then you can create multiple pull requests in parallel.
+3. Make sure that your changes in your fork/branch are tested and render correctly in the Web site.
+4. Submit a pull request from your fork to RSQKit repository (this repository).
+5. If you receive feedback, make changes on your fork and your pull request will update automatically.
+
+Editorial Board Members will review pull requests and have a final say over what gets merged into RSQKit.
+
+> [!NOTE]
+> The reason why we recommend working in your own forks versus in a branch of this repository for collaborators who have push/write
+> permissions to RSQKit repository is primarily because you can render your own forked version of RSQKit as a Web site more easily
+> (which you cannot on the RSQKit repository which hosts a live/production Web site) and test what your changes will look like
+> before creating a pull request. 
+> It also allows reviewers to more easily check and test your changes and make sure everything is correct before merging to the main branch of RSQKit.
+>
+> [Another perspective on why you should fork][github-why-should-i-fork] - this will keep the RSQKit repository in a clean condition, having only official
+> branches and tags and cluttered by a large number of development or feature branches.
+>
+> Finally, working in GitHub interface is convenient, but does not allow for spelling checks and loads of typos creap in this way.
+> If you can - clone your fork and set up RSQKit locally on your machine, and edit files 
+> in a good text editor with spelling checking capabilities.
+
+### What *Not* to Contribute
+
+At this point, we do not welcome contributions that create or change the underlying RSQKit infrastructure - for example, 
+data files in `_data/` folder in the repository, `_config.yml` and other infrastructure files 
+(such as CSS files, page templates and files in the `_includes/` folder).
+If you think some of these files should be updated, please [contact the Editorial Board][contact].
+
+## Contributor responsibilities
 
-## Contibutor responsibilities
 When contributing to this project, please keep in mind the following:
 
-- It is your responsibility to make sure that the content you provide respects copyright.
-  
-- To avoid having to resolve many issues during the revison process, please check the Editor checklist and make sure you have complied with the requirements.
-  
+- It is your responsibility to make sure that the content you provide respects copyright. 
+- To avoid having to resolve many issues during the revision process, please check the Editor checklist and make sure you have complied with the requirements.
```

Comment:
What's the 'Editor checklist' do we have one for RSQKit? or is this a placeholder for the future?